Maple2.Server.Game/Packets/PortalPacket.cs

```diff
@@ -26,18 +27,18 @@ private enum Command : byte {
         pWriter.Write<Vector3>(fieldPortal.Position);
         pWriter.Write<Vector3>(fieldPortal.Rotation);
         pWriter.Write<Vector3>(portal.Dimension);
-        pWriter.WriteUnicodeString(); // Model: Eff_Com_Portal_E
+        pWriter.WriteUnicodeString(fieldPortal.Model);
         pWriter.WriteInt(portal.TargetMapId);
         pWriter.WriteInt(fieldPortal.ObjectId);
-        pWriter.WriteInt();
+        pWriter.Write<PortalActionType>(portal.ActionType);
         pWriter.WriteBool(fieldPortal.MinimapVisible);
-        pWriter.WriteLong();
-        pWriter.WriteByte(portal.Type);
-        pWriter.WriteInt(); // StartTick
+        pWriter.WriteLong(fieldPortal.HomeId);
+        pWriter.Write<PortalType>(portal.Type);
+        pWriter.WriteInt(fieldPortal.EndTick);
```

Comment:
are you sure? previous comment said `StartTick` and the int below is `EndTick`

Reply:
Yep, pretty positive about this. It wouldn't show the duration of the portal until I swapped it to EndTick